### PR TITLE
test(lines-between-class-members): add tests for corner cases

### PR DESCRIPTION
**Pull request by pumano**
### Description
Add valid test cases where long class member formatted via prettier to few lines warn as false positive

 1st test - OK
 2nd test - FAILED

### Additional context
should be fixed to pass tests

### CODE REVIEW

File: packages/eslint-plugin/rules/lines-between-class-members/lines-between-class-members._ts_.test.ts [hunk 1]
```diff
@@ -63,6 +63,52 @@ run<RuleOptions, MessageIds>({
...
+      options: ['always', { exceptAfterOverload: true, exceptAfterSingleLine: true }],
+    },
+    {
+      code: $`
+        class foo {
+        a = 1;
+        b = someFunction(
+          withParametersFormattedViaPrettier,
+        );
+        c = 3;
```
Comment (9romise) [inline packages/eslint-plugin/rules/lines-between-class-members/lines-between-class-members._ts_.test.ts]: Why should this be passed? `b` is not a single-line node.

You can specify [`enforce`](https://eslint.style/rules/lines-between-class-members#enforce) like:
```ts
'@stylistic/lines-between-class-members': [
  'error',
  {
    enforce: [
      { blankLine: 'always', prev: '*', next: '*'},
      { blankLine: 'never', prev: 'field', next: 'field'},
    ]
  },
  {
    exceptAfterOverload: true,
    exceptAfterSingleLine: true,
  }
]
```
